.trivyignore

```diff
@@ -53,3 +53,10 @@ CVE-2023-26115
 # rexml (react-native)
 
 CVE-2024-35176
+CVE-2024-39908
```

Comment:
These require upgrading the react-native package which is no small feat, so ignoring for now. We do have a ticket for the big upgrade https://gr4vy.atlassian.net/browse/TA-6984